pkg/render/logstorage_test.go

```diff
@@ -1626,6 +1627,45 @@ var _ = Describe("Elasticsearch rendering tests", func() {
 			Expect(kibana.Spec.PodTemplate.Spec.Affinity).NotTo(BeNil())
 			Expect(kibana.Spec.PodTemplate.Spec.Affinity).To(Equal(podaffinity.NewPodAntiAffinity("tigera-secure", "tigera-kibana")))
 		})
+
+		It("should render the kibana pod template with resource requests and limits when set", func() {
```

Comment:
Would be good to include the init container as well in this test.

Reply:
Done